xtask/src/cargo.rs

```diff
@@ -277,6 +291,13 @@ impl Cargo {
 
         if let Some(target) = self.target {
             cmd.args(["--target", target.as_triple()]);
+
+            // If the target is not installed, use build-std. Keep this
+            // around until our minimum-supported nightly version is at
+            // least 2022-11-10.
+            if !is_target_installed(target.as_triple())? {
+                cmd.args(["-Zbuild-std=core,alloc"]);
```

Comment:
https://github.com/rust-osdev/uefi-rs/pull/585

can be omited I guess?

Reply:
Oops! Not quite sure how that code snuck back in here, fixed.